Dockerfile.reborn

```
@@ -51,6 +51,10 @@ ENV CARGO_PROFILE_DIST_PANIC=abort \
     CARGO_PROFILE_DIST_CODEGEN_UNITS=1
 
 COPY --from=planner /app/recipe.json recipe.json
+COPY crates/ironclaw_webui_v2/frontend/ crates/ironclaw_webui_v2/frontend/
+RUN cd crates/ironclaw_webui_v2/frontend \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)